docs/decisions/20221006-store-private-keys-of-issuers-inside-prism-agent.md

```diff
@@ -1,30 +1,30 @@
-# Store private keys of Issuers inside prism-agent
+# Store private keys of Issuers inside the cloud-agent
```

Comment:
`inside the Cloud Agent` or `inside cloud-agent`: make a choice on the way to describe

Reply:
thanks, fixed.